### PR TITLE
UserGuide.adoc: Update Event Management section

### PR DESCRIPTION
**Pull request by Kratious**

### CODE REVIEW

**Comment by sharan8** [comment]
Just some minor thoughts: 
1. Should we add a full stop after the end of a bullet point? (Need to standardise)
2. This phrase is used quite widely in the User Guide - maybe the rest of us can update to "when in the 'event' context." or "when in the 'volunteer' context" as well. Currently mine is "when in the volunteers context".

Also liked the use of the table to show the event properties!

**Comment by iMarbles** [comment]
> Just some minor thoughts:
> 
> 1. Should we add a full stop after the end of a bullet point? (Need to standardise)
> 2. This phrase is used quite widely in the User Guide - maybe the rest of us can update to "when in the 'event' context." or "when in the 'volunteer' context" as well. Currently mine is "when in the volunteers context".
> 
> Also liked the use of the table to show the event properties!

1. Agree with the use of full stop after bullet point.
2. Mine uses "when in the volunteers' context". Note the use of single quote for the context name.

**Comment by Kratious** [comment]
1. OK, let's agree on full stops after bullet points
2. I think the single quote is a good way to differentiate the context name

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -194,30 +195,79 @@ Deletes the details of the volunteer specified at index 1
...
 
-* `add n/Flag Day l/Yishun MRT S sd/31-10-2018 ed/31-10-2018 d/For the children's home`
-* `add n/Fundraising l/Tampines S sd/31-09-2018 ed/31-09-2018 d/Raising funds  t/fundraiser t/charity`
+* `add n/Flag Day l/Yishun MRT sd/31-10-2018 ed/31-10-2018 st/09:00 et/15:00 d/For the children's home` +
++
+Adds an event with the following properties:
++
+[options="header"]
+|===
+|Property   |Values
```
Comment (iMarbles) [inline docs/UserGuide.adoc]: Should header be 'value' instead of 'values'?

**Comment by iMarbles** [comment]
Looks good.

**Pull request merged**